Packs/TrendMicroVisionOne/ReleaseNotes/4_1_0.md

```diff
@@ -0,0 +1,20 @@
+
+#### Integrations
+
+##### Trend Micro Vision One Event Collector
+
+##### Trend Micro Vision One V3.
+
+- Added action to run a custom script on an endpoint.
+- Added action to get a list of available custom scripts in XDR Portal.
+- Added action to add a custom script to XDR Portal.
+- Added action to update a custom script in XDR Portal.
+- Added action to download a custom script from XDR Portal.
+- Added action to delete a custom script from XDR Portal.
```

Comment:
```suggestion
- Added new commands:
    - command 1
    - command 2
```
Please use this template, replace command # with the name of the command as mentioned in the yml.

Reply:
Done.